.electron-vue/dev-runner.js

```diff
@@ -1,190 +0,0 @@
-'use strict'
```

Comment:
So this file is for run a dev environment.
I'm sure that you have an alternative to that. I will try to run this PR on my workspace and I will see if we have the right paths and hot-reloading and so on.

Reply:
there is some problem with hot reloading on files under the /src/service/